code/modules/ai/ai_behaviors/carbon/xeno/drone.dm

```diff
@@ -1,17 +0,0 @@
-//A example drone that uses it's weeding ability provided no other weeds are nearby
```

Comment:
so why were these deleted? were they never being compiled in?

Reply:
These were only added in when i didn't have the ``ai_should_start_consider``, because of the new ai consider for abilities, it's now possible for any other xeno to use xeno abilities if it iwas varedited into the ability list (runners using drone lay weeds, drones pouncing etc)

This removes a good amount of hardcoding each xeno to use specific abilities and these files weren't included in the compile